fmc/src/flow/mod.rs

```diff
@@ -30,5 +33,12 @@ use caliptra_drivers::CaliptraResult;
 ///
 /// * `env` - FMC Environment
 pub fn run(env: &mut FmcEnv) -> CaliptraResult<()> {
+    let reset_reason = env.soc_ifc.reset_reason();
+
+    if reset_reason == ResetReason::ColdReset {
+        // Generate the FMC Alias Certificate Signing Request (CSR)
```

Comment:
Was there to be any kind of request that prompts this? I think always doing it was one option. Just want to make sure this is the one we want since there will be a little bit of a performance impact.